.github/00-bug-performance-issue.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug/Performance Issue
+about: Use this template for reporting a bug or a performance issue.
+
+---
+
+<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>
+
+Consider Stack Overflow for getting support using TensorBoard - they have a larger community with better searchability:
+
+https://stackoverflow.com/questions/tagged/tensorboard
+
+**System information**
+- Tensorboard version(from pip package, also printed out when running `tensorboard`) :
+- TensorFlow version if different from TensorBoard :
+- OS Platform and version (e.g., Linux Ubuntu 16.04) :
+- Python version (e.g. 2.7, 3.5) :
+- For browser-related issues :
+  - Browser type and version (e.g. Chrome 64.0.3282.140) :
+  - Screenshot if it's a visual issue :
+
+
+**Describe the current behavior**
+
+**Describe the expected behavior**
+
+**Code to reproduce the issue**
+Please describe the bug as clearly as possible, and if possible provide a minimal example (code, data, and/or command line) to reproduce the issue.
+
+**Other info / logs**
+Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.
```

Comment:
You may want to ask for screenshots too as Tensorboard will have more visual issue.

Reply:
Done